src/Transform/ONNX/ConstProp.cpp

```diff
@@ -226,6 +226,33 @@ Value ConstPropElementwiseUnary(
       .getResult();
 }
 
+//===----------------------------------------------------------------------===//
+// Code to perform constant propagation for ONNXWhereOp in presence of
+// broadcast.
+//
+/// Does element-wise ternary (cond ? lhs : rhs) with broadcast on all inputs.
+//===----------------------------------------------------------------------===//
+
+Value ConstPropWhere(PatternRewriter &rewriter, Value replacingValue,
+    Value condValue, Value lhsValue, Value rhsValue) {
+  ConstPropCounters::count("ElementwiseBinary", {lhsValue, rhsValue});
```

Comment:
I don't think `ElementwiseBinary` is the correct term for this?

Reply:
good catch, fixed